index.html

```diff
@@ -72,12 +72,9 @@ <h2>Blog</h2>
       </article>
       <article id="contact">
         <h2>Contact</h2>
-        <p>
-          djlefae aelfjae flaef aefjlejf aefjl Lorem ipsum dolor sit amet,
-          consectetur adipisicing elit. Cupiditate laudantium, tenetur amet
-          libero minima aut reprehenderit doloremque optio voluptate corporis
-          nostrum qui quod expedita vitae eius error. Vel, neque laboriosam?
-        </p>
+        <a href="">linkedln</a>
+        <a href="">facebook</a>
+        <a href="">twitter</a>
```

Comment:
These `a` tags are not linking anywhere because of the empty `href` attribute, right? In this case, they are useless and I would not recommend using them, but choosing a different HTML element. 

Since it looks like you want to include links here, but if you don't want to use links to your profile pages, better use a generic url for the `href` like `www.linkedin.com` and `www.facebook.com`.